geoips/plugins/yaml/workflows/read_test.yaml

```diff
@@ -0,0 +1,29 @@
+interface: workflows
+family: order_based
+name: read_test
+docstring: |
+  11.2 µm Infrared.
+
+  This product utilizes channel 14 (11.2 µm) and highlights areas of deep convection within a Tropical Cyclone.
+description: 11.2 µm Infrared.
+relpath: geoips/plugins/yaml/workflows/read_test.yaml
+abspath: home/kumar/geoips/geoips/geoips/plugins/yaml/workflows/read_test.yaml
+spec:
+  steps:
+    - reader:
+        name: abi_netcdf
+        arguments:
+          variables: ['B14BT']
+    - algorithm:
+        name: single_channel
+        arguments:
+          variables: ['B14BT']
+          output_data_range: [-90.0, 30.0]
+          input_units: Kelvin
+          output_units: celsius
+          min_outbounds: crop
+          max_outbounds: crop
+          norm: null
+          inverse: null
+    - workflow:
+        name: read_test
```

Comment:
What does this step do?

Reply:
It is a dummy step right now that just tests whether the OBP will attempt to load an embedded workflow. At the moment, this just prints info indicating that it found and loaded the embedded workflow but doesn't actually run it.

The intention of this kind of step is to allow defining stub workflows, similar to ProductDefaults, that can be embedded into other workflows. When called, the plugin would call the OBP again (so, recursive OBP calls) to process the embedded workflow.

Reply:
Ah, the intent is to fully support the current format in addition to the new format. We're starting by getting the design in place, then we intend to write translators for most and, hopefully all, product families. There is another PR in the works to add pydantic validation for the current Product and ProductDefault plugins. We'll continue using the current procflows until we're content that we've managed to achieve backwards compatibility for at least most Product families.